src/bug-filing/common/issue-filing-url-string-utils.ts

```diff
@@ -4,36 +4,35 @@ import { EnvironmentInfo } from '../../common/environment-info-provider';
 import { title } from '../../content/strings/application';
 import { CreateIssueDetailsTextData } from './../../common/types/create-issue-details-text-data';
 
-export class IssueFilingUrlStringUtils {
-    public static footer(environmentInfo: EnvironmentInfo): string {
+export type StringUtils = {
```

Comment:
StringUtils seems too generic a type for this concern. I think something like IssueUrlCreationUtils or something _else_ that indicates the usage of this class is for this particular context would be nice.

Reply:
Update!